src/snovault/elasticsearch/indexer.py

```diff
@@ -143,140 +154,302 @@ def get_related_uuids(request, es, updated, renamed):
     return (related_set, False)
 
 
+def _determine_indexing_protocol(request, uuid_count):
+    remote_indexing = asbool(request.registry.settings.get('remote_indexing', False))
+    if not remote_indexing:
+        return False
+    default_threshold = 10000
+    try:
+        remote_indexing_threshold = int(
+            request.registry.settings.get(
+                'remote_indexing_threshold',
+                default_threshold,
+            )
+        )
+    except:
```

Comment:
Should try to avoid bare `except`. Looks like here you are trying to check if the conversion fails with `ValueError`, unless there is some failure mode of `request.registry.settings.get` I am missing:
```suggestion
    except ValueError:
```